www/artists/mbolen/artist.json

```diff
@@ -3,5 +3,6 @@
   "location": "Humboldt",
   "changelog": [
     "Created by Connor Brezinsky: 2015."
```

Comment:
You forgot a comma here. This is causing the build to fail, preventing it from performing any of our tests. We can't accept this pull request until the comma is added.

Reply:
I added the commas.